pkg/controller/controller.go

```diff
@@ -298,6 +313,19 @@ func (c *Controller) renderBadRequest(w http.ResponseWriter, m string) {
 	})
 }
 
+func getEmail(iapEmailValue string) (string, error) {
+	if iapEmailValue == "" {
+		return "", fmt.Errorf("email header is not valid")
+	}
+
+	split := strings.Split(iapEmailValue, ":")
+	if len(split) != 2 {
+		return "", fmt.Errorf("email value has unexpected format")
```

Comment:
Care to add want vs. got?

Reply:
Yes, expected format. That's what I mean.

Reply:
i believe i got it to work 

```
	got, want := strings.Split(iapEmailValue, ":"), []string{iapHeaderName + "<email>"}
	if len(got) != 2 {
		return "", fmt.Errorf("email %q does not match expected email pattern %q", got, want)
	}
```